follower/consensus_follower.go

```diff
@@ -43,9 +45,30 @@ func WithBootstrapDir(bootstrapDir string) Option {
 	}
 }
 
+// BootstrapNodeInfo contains the details about the upstream bootstrap peer the consensus follower uses
+type BootstrapNodeInfo struct {
```

Comment:
@smnzhu as per earlier comment, added this new struct instead of the two arrays of strings.